src/store/actions.ts

```diff
@@ -465,13 +466,16 @@ export const resubmit: any = async ({ commit, getters }): Promise<boolean> => {
   // commit the original NR's data
   commitExistingData({ commit, getters })
 
+  // override the request action code
+  commit('mutateRequestAction', RequestCode.RESUBMIT)
+
   // build the request data
   const nrTypeData = getNrTypeData({ getters }, NrType.DRAFT)
 
-  // override resubmission message in Additional Info field
-  nrTypeData['additionalInfo'] = '*** Resubmission of ' + nrData['nrNum'] + ' ***'
+  // add resubmit NR number (for internal use only)
+  nrTypeData['resubmitNrNum'] = nrData['nrNum']
```

Comment:
This new property actually becomes part of the POST payload. The API can either use it or ignore it.